app/tsconfig.json

```
@@ -12,6 +12,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/pages/review", "src/pages/review"],
```

```suggestion
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
```
I don't think we need these anymore since we are no longer added a folder to the authoring folder, but correct me if I'm wrong.